ci_smoke_test.sh

```diff
@@ -16,15 +16,30 @@ mv terraform .tf_dcos_core/terraform
 cd .tf_dcos_core
 export TERRAFORM_PATH=${PWD}/terraform
 ${TERRAFORM_PATH} init
-for dir in ./dcos-versions/*
+# Open DC/OS Check
+for dir in ./open/dcos-versions/*
```

Comment:
I would combine both loops into one

Reply:
The new behavior would look like this:
```
${TERRAFORM_PATH} validate -var "dcos_type=open" -var "role=${role}" -var "dcos_version=${version}"
```
Where a user would need to specify which `dcos_type` they would perfer. By default it is set to open. When the loop checks, it would specifically check it in a single loop. If it checks the EE contents, it will fail.